doc/contributing/hw/comportability/README.md

```diff
@@ -204,7 +204,7 @@ Items on the inout list of the form `name` incur all three.
 
 #### Multiplexing Feature and Pad Control
 
-In the top level chip framework there is a [pin multiplexing unit (`pinmux`)](../../../../hw/ip_templates/pinmux/README.md) which provides flexible assignment to/from peripheral IO and chip pin IO.
+In the top level chip framework there is a pin multiplexing unit (`pinmux`), for example the [Earlgrey pinmux](../../../../hw/top_earlgrey/ip_autogen/pinmux/README.md), which provides flexible assignment to/from peripheral IO and chip pin IO.
```

Comment:
Readability: it is probably clearer to have an absolute path to "hw" than seeing a bunch of ..'s.